app/views/layouts/application.html.erb

```diff
@@ -1,28 +1,32 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>KhanBurmese</title>
-  <%= favicon_link_tag '/favicon.ico' %>
+  <title>KhanAcademy Myanmar</title>
+  <%= favicon_link_tag '/favicon.png' %>
   <%= stylesheet_link_tag    "application", :media => "all" %>
   <%= javascript_include_tag "application" %>
   <%= csrf_meta_tags %>
+  <meta name="viewport" content="width=500, initial-scale=1">
 </head>
 <body>
 
 <!-- Navigation bar. -->
-<nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
+<nav class="navbar navbar-inverse navbar-fixed-top khan-nav" role="navigation">
   <div class="container">
-    <div>
-      <%= image_tag '/favicon.ico', :class => 'img-responsive img-logo' %>
-      <%= link_to 'KhanAcademy Myanmar', root_path, :class => 'navbar-brand pull-left inline-brand' %>
-    </div>
-    <div class="collapse navbar-collapse">
+    <%= link_to(root_path) do %>
+      <%= image_tag('/favicon.png', :class => 'img-responsive img-logo') %>
+      <div class="navbar-brand"><b>Khan</b>Academy <b><i>Myanmar</i></b></div>
+    <% end %>
+    <div class="pull-right">
```

Comment:
You don't need this line. The list class `navbar-right` automatically does this.